src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -236,7 +236,7 @@ ucs_status_t uct_rdmacm_cm_ep_resolve_cb(uct_rdmacm_cm_ep_t *cep)
 
     args.field_mask = UCT_CM_EP_RESOLVE_ARGS_FIELD_DEV_NAME;
     uct_rdmacm_cm_id_to_dev_name(cep->id, args.dev_name);
-    return uct_cm_ep_resolve_cb(cep->super.user_data, &args);
+    return uct_cm_ep_resolve_cb(&cep->super, &args);
```

Comment:
nice :)
how did the previous PR pass the CI..?

Reply:
UCT tests dont use new API directly for now, porting is in progress